linux/keyman-config/km-package-install

```diff
@@ -87,10 +87,13 @@ def list_keyboards():
 def main():
     parser = argparse.ArgumentParser(
       description='Install a Keyman keyboard package, either a local .kmp file or specify a ' +
-      'keyboard id to download and install')
+      'keyboard id to download and install, optionally specifying a language for which to ' +
+      'install the keyboard.',
+      epilog='Example: km-package-install -s -p sil_el_ethiopian_latin --bcp47 ssy-latn')
     parser.add_argument('-s', '--shared', action='store_true', help='Install to shared area /usr/local')
     parser.add_argument('-f', '--file', metavar='KMPFILE', help='Keyman kmp file')
     parser.add_argument('-p', '--package', metavar='PACKAGE', help='Keyman package id')
+    parser.add_argument('--bcp47', metavar='LANGTAG', help='bcp47 language tag')
```

Comment:
```suggestion
    parser.add_argument('-l', '--bcp47', metavar='LANGTAG', help='bcp47 language tag')
```

Suggest the shorthand `-l` is useful because it's easier to remember than `--bcp47`?

Reply:
done